index.test.js

```diff
@@ -87,6 +90,15 @@ describe('validate-commit-msg.js', function() {
       expect(logs).to.deep.equal([msg]);
     });
 
+    it('should log the helpMessage on invalid commit messages', function() {
+      var msg = 'invalid message';
+      m.config.helpMessage = '\nPlease try:\nnpm install commitizen -g && git cz\n';
```

Comment:
I would prefer to avoid encouraging people to install commitizen globally. There are a lot of problems with globally installed modules. I would rather this say:

```
Please fix your commit message (and consider using http://npm.im/commitizen)
```

Or something like that.